data/events.txt

```diff
@@ -798,6 +798,7 @@ event "start of hostilities"
 	government "Free Worlds"
 		"attitude toward"
 			"Republic" -.2
+	"reputation: Republic" = -10
```

Comment:
This should be `<?=` rather than `=` as `=` provides a free rep boost if the player has done things that would otherwise decrease their reputation.

Reply:
It copies the existing functionality.  I'm happy to change it though, this makes sense.